### PR TITLE
NetApp ONTAP module for manage ipspaces

### PR DESCRIPTION
**Pull request by vicmunoz**
##### SUMMARY
New module: na_ontap_ipspace (creation / deletion / rename of IPspaces)

##### ISSUE TYPE
New Module Pull Request

##### COMPONENT NAME
na_ontap_ipspace

### CODE REVIEW

**Review by lonico — APPROVED** [review]
Looking great!

It would be nice if you could add a Unit Test.
There is an example in
https://github.com/ansible/ansible/pull/49784

But this is only a suggestion.

File: test/units/modules/storage/netapp/test_na_ontap_ipspace.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+from ansible.modules.storage.netapp.na_ontap_ipspace \
+    import NetAppOntapIpspace as my_module
+from units.compat import unittest
+from units.compat.mock import patch
+
+if not netapp_utils.has_netapp_lib():
+    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```
Comment (lonico) [inline test/units/modules/storage/netapp/test_na_ontap_ipspace.py]: Can you change pytest.skip to pytest.mark.skip  ?

(We'll need to change it in all modules).

**Review by lonico — APPROVED** [review]

File: test/units/modules/storage/netapp/test_na_ontap_ipspace.py (hunk 1 above)
Comment (vicmunoz) [reply test/units/modules/storage/netapp/test_na_ontap_ipspace.py]: updated

**Review by lonico — APPROVED** [review]

**Review by lonico — APPROVED** [review]
shipit

**Review by lonico — APPROVED** [review]
shipit

**Review by carchi8py — APPROVED** [review]
Shipit

**Review by lonico — APPROVED** [review]
shipit

**Review by lonico — APPROVED** [review]
shipit

The bot failed with:
lib/ansible/modules/storage/netapp/na_ontap_ipspace.py:0:0: E307 version_added should be '2.9'. Currently '2.8'

But in my opinion, this should go into 2.8.   This PR has been opened for a very long time.
Why does it take so long to get any feedback from Red Hat?

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — COMMENTED** [review]
lgtm, thanks for adding in the tests.

**Pull request merged**